ProfileView/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@cjaas/profile-view-widget",
+  "version": "1.0.0",
+  "author": {
+    "name": "Kevin Hyde",
+    "email": "kehyde@cisco.com"
+  },
+  "license": "MIT",
+  "repository": "https://github.com/CiscoDevNet/cjaas-widgets",
+  "main": "./dist/index.js",
+  "types": "./dist/types/index.d.ts",
+  "scripts": {
+    "start": "npm run dev",
+    "dev": "npm run check && webpack-dev-server --config ./webpack.config.dev.server.ts",
+    "dev:build": "npm run check && webpack --config-name=dev",
+    "dist": "npm run check && webpack --config-name=distProd",
+    "dist:dev": "npm run check && webpack --config-name=distDev",
+    "dist:watch": "npm run check && webpack --config-name=distDevWatch",
+    "npm:auth": "uuip-publish-auth [agentx,uuip,momentum-ui-private,wxcc-desktop]",
+    "npm:publish": "npm run npm:auth && npm run dist && uuip-publish dist agentx",
+    "check": "check-peer-dependencies --yarn",
+    "test": "jest"
+  },
+  "resolutions": {
+    "jest": "^26.4.2",
+    "jest-config": "^26.4.2",
+    "lit-element": "2.3.1",
+    "lit-html": "1.2.1"
+  },
+  "peerDependencies": {
+    "@momentum-ui/core": "^19.9.12",
+    "@momentum-ui/icons": "^7.54.0",
+    "@momentum-ui/utils": "^6.2.7",
+    "@momentum-ui/web-components": "^2.1.2",
+    "lit-element": "2.3.1",
+    "lit-html": "1.2.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/preset-env": "^7.9.0",
+    "@cjaas/common-components": "^1.0.0-dev.4",
```

Comment:
oh i had no idea you guys were publishing so much :) what were the changes?

Reply:
oh shoot . . . hang on I was trying something at Elena's request. I'm resetting to basic release